### DISCUSSION TITLE
Is Spam checking enabled by default?

### DISCUSSION THREAD

**Discussion by binarydad** [modoboa/modoboa#2388] [Help]
I've just installed 1.17.0 after moving from MiaB and am not entirely sure if spam checks (such as Spamassassin) are enabled. Previously, I would see `X-Spam-***` headers in emails on MiaB, but do not see those on Modoboa. I see it's running `X-Virus-Scanned` for amavisd, but nothing explicitly for spam. 

I see no errors logged, but I see nothing regarding spam checks in mail.log. 

How can I ensure spam checks are happening and enable the headers in email? Thanks.

**Comment by tonioo** [comment]
@binarydad By default, modoboa comes with amavis/spamassassin/clamav filters. If clamav runs, then it means that amavis is running fine and so is spamassassin. But you might have to tweak default scores a bit according to your needs
